Sources/PersistentCollections/Node/_Stack.swift

```diff
@@ -0,0 +1,115 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+/// A fixed-size array of just enough size to hold an ancestor path in a
+/// `PersistentDictionary`.
+@usableFromInline
+@frozen
+internal struct _Stack<Element> {
+#if arch(x86_64) || arch(arm64)
+  @inlinable
+  @inline(__always)
+  internal static var capacity: Int { 13 }
+
+  @usableFromInline
```

Comment:
For consistency with `#else`?

```suggestion
  // xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxxx xxxx
  @usableFromInline
```

Reply:
Oh oops I didn't mean to commit that doodle actually -- I just wanted to prove to myself the values are right. But it does seem like a good visual, so let's do it